node/src/GlideClusterClient.ts

```diff
@@ -754,12 +756,16 @@ export class GlideClusterClient extends BaseClient {
      * ```
      */
     public async fcallWithRoute(
-        func: string,
-        args: string[],
-        route?: Routes,
+        func: GlideString,
+        args: GlideString[],
+        options?: {
+            route?: Routes;
+            decoder?: Decoder;
+        },
```

Comment:
```suggestion
        options?: RouteOption & DecoderOption
```
Please see #2160 and #2179 for an example. Update doc accrodingly.